apps/wallet-mobile/src/components/KeyboardAvoidingView/KeyboardAvoidingView.tsx

```diff
@@ -20,8 +18,6 @@ export const KeyboardAvoidingView = ({
       {...rest}
     >
       {children}
-
-      {Platform.OS === 'android' && !enabled && <Space height="lg" />}
```

Comment:
dropped since more than one keyboard-view screen can be on at same time (confirmation + modal with input)